README.md

```diff
@@ -21,10 +21,25 @@ Here's the decisions we taken so far:
 | ADR | Title                         | PR (discussion) |
 | --- | ----------------------------- | --------------- |
 | 1   | [Record architecture decisions](doc/architecture/decisions/0001-record-architecture-decisions.md) | [PR#25](https://github.com/teamdigitale/italia-pagopa-proxy/pull/25)                |
-| 2   | [Use uuid/v1 for sequential requests](doc/adr/0002-use-uuid-v1-for-sequential-requests.md) | [PR#8](https://github.com/teamdigitale/italia-pagopa-proxy/pull/8)                |
+| 2   | [Use uuid/v1 for sequential requests](doc/adr/0002-use-uuid-v1-for-sequential-requests.md) | [PR#37](https://github.com/teamdigitale/italia-pagopa-proxy/pull/37)                |
 
 ## NOTES
 
 This project includes @types/bluebird because it's necessary for node-soap.
 It will be removed when this dependency will be fixed into soap:
-https://github.com/vpulim/node-soap/pull/1009
\ No newline at end of file
+https://github.com/vpulim/node-soap/pull/1009
+
+### Environment variables
+
+Those are all Environment variables needed by the application:
+
+| Variable name                          | Description                                                                       | type   |
+|----------------------------------------|-----------------------------------------------------------------------------------|--------|
+| WINSTON_LOG_LEVEL                      | The log level used for Winston logger                                             | string |
```

Comment:
this is not a generic string, you can't put whatever you want there, please provide a list of valid values